kroxylicious-kms-provider-aws-kms/src/main/java/io/kroxylicious/kms/provider/aws/kms/model/DecryptResponse.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.aws.kms.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@SuppressWarnings("java:S6218") // we don't need DecryptResponse equality
+public record DecryptResponse(@JsonProperty(value = "KeyId") String keyId,
+                              @JsonProperty(value = "Plaintext") byte[] plaintext) {
```

Comment:
how well do byte arrays actually go in JSON / YAML? do they actually need base64 encoded or something?

Reply:
Jackson takes care of the serialising the byte array to the base64 for us.

<img width="1482" alt="Screenshot 2024-05-16 at 16 15 21" src="https://github.com/kroxylicious/kroxylicious/assets/18440250/5fef6a01-00a8-49d6-8236-fb5876431e49">

Reply:
Ah nice.